src/lexer.rs

```diff
@@ -360,7 +360,7 @@ impl Lexer {
             }
             Some(_) => match self.read_until(false, false, false, Box::new(is_token_split)) {
                 Ok(w) => {
-                    println!("The words I got: {:?}", w);
+                    debug_println!("The words I got: {:?}", w);
```

Comment:
Whoops, this print statement did slid past me before I pushed. Thanks for catching it. 

Honestly I'm not sure whether something like this makes sense as a `debug_println`, or if it should just be removed.

Reply:
Heheh, ok. I've left it in for now, feel free to remove it later.